### DISCUSSION TITLE
将json 通过工具转换为pag之后，帧率变低了，看起来变卡了

### DISCUSSION THREAD

**Discussion by carleft** [Tencent/libpag#1956] [Q&A] [answered]
视频中左边是lottie，右边是对应的pag
电脑端预览能看出pag明显卡顿，放在性能不好的Android设备上更加明显

https://github.com/Tencent/libpag/assets/33175359/5a2accbd-6836-4670-9111-6a41aac901f8

nomal_mode_icon.json](https://github.com/Tencent/libpag/files/13575226/nomal_mode_icon.json)
[nomal_mode_icon.zip](https://github.com/Tencent/libpag/files/13575230/nomal_mode_icon.zip)
 pag文件是json通过 [PAGConvertor](https://pag.art/file/PAGConvertor_mac_1.4.zip) 工具转换得到的

**Comment by carleft** [comment]
电视上表现效果更明显，左边是lottie， 右边是pag


https://github.com/Tencent/libpag/assets/33175359/d009b0a6-2274-4f6c-9453-c18e801586cd

**Comment by hubeishuaige** [comment]
怎么把json文件转成 pag文件？PAGConvertor 这个API没有找到，谢谢

**Comment by domchen** [comment, marked as answer]
你这个是因为对比的根本不是一个分辨率。右边的是全屏播放，有好几倍的性能压力。

**Reply by Essentiallove** [reply]
大佬，我有个问题，使用含有base64图片的lottie json文件转换为pag时会失败，不含有图片（只有形状，转换工具示例的json就是不含图片的） 的json可以转换，还有我用转换工具示例的apng和序列帧导出也会失败，失败提示都是打不开文件，这个是需要有什么配置吗？
